lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -55,7 +55,8 @@
                         })
                     );
                     this.discussionView = new TeamDiscussionView({
-                        el: this.$('.discussion-module')
+                        el: this.$('.discussion-module'),
+                        readOnly: this.$('.discussion-module').data('read-only')
```

Comment:
I recommend changing this value to be a boolean, rather than the string. I think this will make the rest of the code simpler to understand. So, change this line to:

    readOnly: this.$('.discussion-module').data('read-only') === 'true'